### PR TITLE
adding Fbx loader

### PR DESCRIPTION
**Pull request by yamahigashi**
Adding the FBXLoader ( [former repo](https://github.com/yamahigashi/FBXLoaderJS) ).

FBX Features Implemented:
Format: FBX version 7.x ascii
Meshes: Indexed Vertices, Trianges/Quadrangle (faces), Normals and UVs
Skinning: SkinIndices and SkinWeights, Bind Pose
Animation

FBX Features NOT Implemented:
NURBS, patch, color of vertex, smoothing groups, Materials, Textures, Cluster, Shape, Chain IK and constraints, multiple takes of animation

### CODE REVIEW

File: examples/webgl_loader_fbx.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+					geometry.vertices.push( new THREE.Vector3( - size, - 0.04, i ) );
+					geometry.vertices.push( new THREE.Vector3(   size, - 0.04, i ) );
+
+					geometry.vertices.push( new THREE.Vector3( i, - 0.04, - size ) );
+					geometry.vertices.push( new THREE.Vector3( i, - 0.04,   size ) );
+
+				}
+
+				var line = new THREE.LineSegments( geometry, material );
```
Comment (WestLangley) [inline examples/webgl_loader_fbx.html]: Consider using something like
```javascript
var helper = new THREE.GridHelper( 14, 1 );
helper.setColors( 0x303030, 0x303030 );
helper.position.set( 0, - 0.04, 0 );
```

File: examples/webgl_loader_fbx.html [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.setClearColor( 0x000000 );
+                container.appendChild( renderer.domElement );
+
+                //////////////////////////////////////////////////////////////
+                // controls, camera
+                controls = new THREE.OrbitControls( camera, renderer.domElement );
+                controls.target.set( 0, 12, 0 );
+				camera.position.set( 2, 18, 28 );
+				camera.lookAt( new THREE.Vector3( 0, 15, 0 ) );
```
Comment (WestLangley) [inline examples/webgl_loader_fbx.html]: `lookAt()` should not be required, but if you call it, then call `lookAt( controls.target )`.

File: examples/webgl_loader_fbx.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+                if( mixers.length > 0 ){
+                    for ( var i=0; i < mixers.length; i++ ){
+                        mixers[i].update( clock.getDelta() );
+
+                    }
+
+                }
+
+				controls.update();
```
Comment (WestLangley) [inline examples/webgl_loader_fbx.html]: `contols.update()` should not be required for `OrbitControls` unless `enableDamping` is true.

File: examples/webgl_loader_fbx.html [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+					geometry.vertices.push( new THREE.Vector3( i, - 0.04,   size ) );
+
+				}
+
+				var line = new THREE.LineSegments( geometry, material );
+				scene.add( line );
+
+                //////////////////////////////////////////////////////////////
+				// Lights
+                scene.add( new THREE.AmbientLight( 0xcccccc ) );
```
Comment (WestLangley) [inline examples/webgl_loader_fbx.html]: Your model has `Basic` material, so lights have no effect. Maybe you want to change that?

**Comment by WestLangley** [comment]
Thanks. Made a few suggestions...  : - )

**Comment by yamahigashi** [comment]
@WestLangley Thanks for your reply and suggestions!

**Comment by WestLangley** [comment]
I think you need to add `webgl_loader_fbx.html` to `examples/files.js`.

**Comment by mrdoob** [comment]
Many thanks!

**Pull request merged**